lib/ts/logger.ts

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { package_version as version } from "./version";
+
+const SUPERTOKENS_DEBUG_NAMESPACE = "com.supertokens";
+
+let __supertokensWebsiteLogging = false;
+
+export function enableLogging() {
+    __supertokensWebsiteLogging = true;
+}
+
+export function logDebugMessage(message: string) {
+    if (__supertokensWebsiteLogging) {
+        console.log(
```

Comment:
where will this be logged? On the browser console? Given that users will be using an emulator or a real device, how will they see the logs?

Reply:
These logs will be visible in the terminal, I tested it myself with expo and have added a screenshot in PR description. It would also work with `react-native` cli as well. Reference: https://stackoverflow.com/a/30135863